_includes/markdown/React.md

```diff
@@ -33,6 +33,18 @@ If you already have a Node / NPM / Yarn project that you wish to include React a
 
 Using the CLI tool is ideal if you are creating a full React App with an API backend.
 
+## Project Folder Organisation
+
+There are many guides on optimal folder organisations for a project. Dan Abramov, creator of Redux and React core team member has [advised the following practise](https://mobile.twitter.com/dan_abramov/status/1027245759232651270):
```

Comment:
1. spelling: *practice
2. Let's not link to tweets with someone's opinion, let try and find some documentation somewhere or write up our own